lib/connect.ts

```diff
@@ -106,9 +106,9 @@ var KiteConnect = function(params) {
         "orders": "/orders",
         "trades": "/trades",
         "order.info": "/orders/{order_id}",
-        "order.place": "/orders/{variety}",
-        "order.modify": "/orders/{variety}/{order_id}",
-        "order.cancel": "/orders/{variety}/{order_id}",
+        "order.place": "/orders/{constiety}",
```

Comment:
This also got replaced while replacing `var` with `const`

Reply:
Done!